src/sentry/tasks/deliver_from_outbox.py

```diff
@@ -88,19 +91,20 @@ def schedule_batch(
             # Notably, when l and h are close, this will result in creating tasks that are processing future ids --
             # that's totally fine.
             for i in range(concurrency):
-                drain_task.delay(
-                    outbox_name=outbox_name,
-                    outbox_identifier_low=lo + i * batch_size,
-                    outbox_identifier_hi=lo + (i + 1) * batch_size,
-                )
+                if hasattr(drain_task, "delay"):
+                    drain_task.delay(
+                        outbox_name=outbox_name,
+                        outbox_identifier_low=lo + i * batch_size,
+                        outbox_identifier_hi=lo + (i + 1) * batch_size,
+                    )
 
             deepest_shard_information = outbox_model.get_shard_depths_descending(limit=1)
             max_shard_depth = (
                 deepest_shard_information[0]["depth"] if deepest_shard_information else 0
             )
             metrics.gauge(
                 "deliver_from_outbox.maximum_shard_depth",
-                value=max_shard_depth,
+                value=float(max_shard_depth),
```

Comment:
I think instead this should change the `0` above to `0.0` such that it is treated as a float